participants/ikabansal04.md

```diff
@@ -3,3 +3,20 @@ Github Username: ikabansal04
 University: Indira Gandhi Delhi Technical University for Women
```

Comment:
Maybe you would like to separate these into different lines? You can view your file [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/fffb3220549d02b9b2a5ba35feb048d3e6b09f4b/participants/ikabansal04.md).